### PR TITLE
Update booteeprom.md

### PR DESCRIPTION
**Pull request by timg236**
@lurch 
Work in progress - make this page easier to understand and rework to use the new rpi-eeprom-config --edit and --apply options

* Remove out of date statements about USB MSD
* Remove section about why there is an SPI EEPROM - it was only relevant at launch
* Describe the --edit and --apply arguments
* Describe the friendly commands first

### CODE REVIEW

File: hardware/raspberrypi/booteeprom.md [hunk 1]
```diff
@@ -2,82 +2,96 @@
...
-To check that the bootloader is working correctly, turn off the power, unplug everything from the Raspberry Pi 4, including the SD card, and then turn the power back on. If the green LED blinks with a repeating pattern then the bootloader is running correctly, and indicating that `start*.elf` has not been found. Any other actions imply that the bootloader is not working correctly and should be reinstalled using `recovery.bin`.
-
-## Recovery image
-
-If the Raspberry Pi is not booting it's possible that the bootloader EEPROM is corrupted. This can easily be reprogrammed using the Raspberry Pi Imager tool which is available via the [raspberrypi.org downloads page](https://www.raspberrypi.org/downloads/).
+## Updating the bootloader
 
-Using the recovery image will erase any custom configuration options, resetting the bootloader back to factory defaults.
+### Raspberry Pi Imager
+The easiest way to to update the bootloader to the latest version with default settings is to use the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to install a boot recovery image on to a spare SD card.
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:8]: on to -> onto

You might also want a quick sentence to state that Raspberry Pi OS will automatically keep the EEPROM up-to-date, but it only does the actual update at boot time, e.g.

`Raspberry Pi OS also keeps the EEPROM up-to-date: new EEPROM images are applied when the Raspberry Pi 4 next boots.`

File: hardware/raspberrypi/booteeprom.md [hunk 2]
```diff
@@ -2,82 +2,96 @@
...
+vcgencmd bootloader_version
+```
 
-Note that by default, updating the bootloader (automatically or manually) will retain any custom configuration options of the previous installed version. You can override the migration by manually updating and using the `-d` option with `rpi-eeprom-update`. This will force the updated bootloader to use its inbuilt defaults.
+## Updating the EEPROM configuration
+The bootloader EEPROM image contains a configuration file to define the boot behaviour (e.g. selecting between SD, network and USB boot). 
+The `rpi-eeprom-config` tool may be used to modify the configuration within an EEPROM image file. 
 
-## Manually checking if an update is available
+See the [Bootloader Configuration Page](./bcm2711_bootloader_config.md) for details of the of the configuration file.
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:47]: `of the of the` -> `of the`

File: hardware/raspberrypi/booteeprom.md [hunk 3]
```diff
@@ -2,82 +2,96 @@
...
 
-Running the `rpi-eeprom-update` command with no parameters indicates whether an update is required. An update is required if the timestamp of the most recent file in the firmware directory (normally `/lib/firmware/raspberrypi/bootloader/critical`) is newer than that reported by the current bootloader.
-The images under `/lib/firmware/raspberrypi/bootloader` are part of the `rpi-eeprom` package and are only updated via `apt update`.
+### Reading the current EEPROM configuration
+To view the configuration file used by the bootloader at boot time run `rpi-eeprom-config` or `vcgencmd bootloader_config`
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:50]: Missing full stop.

File: hardware/raspberrypi/booteeprom.md [hunk 4]
```diff
@@ -2,82 +2,96 @@
...
+
+### Editing the current bootloader configuration
+The following command loads the current EEPROM configuration into a text editor. When the editor is closed `rpi-eeprom-config` applies the updated configuration to latest available EEPROM release and uses `rpi-eeprom-update` to schedule an update when the system is rebooted
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:53]: I would put a comma after `When the editor is closed` - before `rpi-eeprom-config`

File: hardware/raspberrypi/booteeprom.md [hunk 5]
```diff
@@ -2,82 +2,96 @@
...
 
 ```
-sudo rpi-eeprom-update
+sudo rpi-eeprom-config --edit
+sudo reboot
 ```
+If the updated configuration file is identical or empty then the update is cancelled.
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:59]: Might be more appropriate to say that `no changes are made` rather that `the update is cancelled`.

File: hardware/raspberrypi/booteeprom.md [hunk 6]
```diff
@@ -2,82 +2,96 @@
...
 
-If an update is available, you can install it using :
-
+### Applying a saved configuration file
+The following command applies `boot.conf` to the latest available EEPROM image and uses `rpi-eeprom-update` to schedule an update when the system is rebooted.
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:62]: Colon at end of line instead of full stop.

File: hardware/raspberrypi/booteeprom.md [hunk 7]
```diff
@@ -2,82 +2,96 @@
...
+To read the configuration file from an EEPROM image file:
+```
+rpi-eeprom-config pieeprom.bin
+```
+
+### Low level commands
 
-To view the configuration file used by the bootloader at boot time
+#### Updating the configuration in an EEPROM image
+The following command reads `pieeprom.bin` and replaces the configuration file with the contents of `boot.conf`. The result is written to `new.bin`
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:77]: Colon at end of line

File: hardware/raspberrypi/booteeprom.md [hunk 8]
```diff
@@ -2,82 +2,96 @@
...
 ```
-vcgencmd bootloader_config
+rpi-eeprom-config --config boot.conf --out new.bin pieeprom.bin
 ```
 
-### Reading the EEPROM version
+For more information about advanced options please run `rpi-eeprom-config -h`
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:82]: Full stop required.

File: hardware/raspberrypi/booteeprom.md [hunk 9]
```diff
@@ -2,82 +2,96 @@
...
+
+#### Updating the bootloader EEPROM
+The following will cause the bootloader EEPROM to be updated the next time the system is rebooted.
 ```
-vcgencmd bootloader_version
+# -d instructs rpi-eeprom-update to use the EEPROM configuration inside new.bin instead of
+# of replacing it with the current configuration from vcgencmd bootloader.
+sudo rpi-eeprom-update -d -f new.bin
 ```
+For more information about advanced options please run `rpi-eeprom-update -h`
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:91]: Full stop

**Review by andrum99 — COMMENTED** [review]
Some minor things.

File: hardware/raspberrypi/booteeprom.md (hunk 2 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:47]: done

File: hardware/raspberrypi/booteeprom.md (hunk 1 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:8]: done

File: hardware/raspberrypi/booteeprom.md (hunk 3 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:50]: done

File: hardware/raspberrypi/booteeprom.md (hunk 4 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:53]: make sense, added both

File: hardware/raspberrypi/booteeprom.md (hunk 6 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:62]: done

File: hardware/raspberrypi/booteeprom.md (hunk 8 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:82]: done

File: hardware/raspberrypi/booteeprom.md (hunk 5 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:59]: Good point, actually, if you make and edit then cancel a subsequent edit without a reboot then the it won't actually go and delete the previous pending update.

File: hardware/raspberrypi/booteeprom.md [hunk 10]
```diff
@@ -2,82 +2,118 @@
...
+At power on, the BCM2711 ROM looks for a file called `recovery.bin` in the root directory of the boot partition on the sd-card. If a valid `recovery.bin` is found then the ROM executes this instead of the SPI EEPROM image. This mechanism ensures that the bootloader SPI EEPROM can always be reset to valid image with factory default settings.
+
+#### EEPROM update files
+| Filename | Purpose |
+|----------|---------|
+| recovery.bin | VideoCore EEPROM recovery executable |
+| pieeprom.upd | Bootloader EEPROM image |
+| pieeprom.bin | Bootloader EEPROM image (same as pieeprom.upd) |
+| pieeprom.sig | The sha256 checksum of bootloader image (pieeprom.upd/pieeprom.bin) |
+| vl805.bin | The VLI805 USB firmware EEPROM image. Pi 4 revision 1.4 and newer do not have a dedicated SPI EEPROM for the VL805 USB controller |
```
Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:110]: It's not clear from that whether the vl805.bin file is required for R6 / v1.4 boards. Is this meant to indicate that the file will only be present when rpi-eeprom-update has been run on an earlier board, i.e. with the discrete EEPROM for the VL805?

Comment (andrum99) [inline hardware/raspberrypi/booteeprom.md:110]: It's not clear from that whether the `vl805.bin` file is required for R6 / v1.4 boards. Is this meant to indicate that the file will only be present when `rpi-eeprom-update` has been run on an earlier board, i.e. with the discrete EEPROM for the VL805?

File: hardware/raspberrypi/booteeprom.md [hunk 11]
```diff
@@ -2,82 +2,118 @@
 
 The Raspberry Pi 4 has an SPI-attached EEPROM (4MBits/512KB), which contains code to boot up the system and replaces `bootcode.bin` previously found in the boot partition of the SD card. Note that if a `bootcode.bin` is present in the boot partition of the SD card in a Pi 4, it is ignored.
 
-## Why use an SPI EEPROM?
+## Boot diagnostics
+If an error occurs during boot then an [error code](../../configuration/led_blink_warnings.md) will be displayed via the green LED. Newer versions of the bootloader will display a [diagnostic message](boot_diagnostics.md) will be shown on both HDMI displays.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:6]: "which will be shown" ?

File: hardware/raspberrypi/booteeprom.md [hunk 12]
```diff
@@ -2,82 +2,118 @@
...
-
 ```
 sudo apt update
 sudo apt full-upgrade
 sudo reboot
 ```
 
-If you wish to control when the updates are applied you can disable the systemd service from running automatically and run `rpi-eeprom-update` manually, as shown in the 'Manually checking if an update is available' section below.
+## Manually checking if an update is available
+Running the `rpi-eeprom-update` command with no parameters indicates whether an update is required. An update is required if the timestamp of the most recent file in the firmware directory (normally `/lib/firmware/raspberrypi/bootloader/critical`) is newer than that reported by the current bootloader.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:30]: Might be worth clarifying that this "timestamp" is embedded inside the file itself, and isn't the last-modified timestamp of the file? Alternatively you could just replace "timestamp" with "version"?

Perhaps "in the firmware directory" should be "in the relevant firmware directory" ?

File: hardware/raspberrypi/booteeprom.md [hunk 13]
```diff
@@ -2,82 +2,118 @@
...
+The images under `/lib/firmware/raspberrypi/bootloader` are part of the `rpi-eeprom` package and are only updated via `apt update`.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:31]: Technically it's `apt upgrade` that updates the `rpi-eeprom` package - `apt update` just updates the lists-of-available-packages :wink:

File: hardware/raspberrypi/booteeprom.md [hunk 14]
```diff
@@ -2,82 +2,118 @@
...
 
-The `FREEZE_VERSION` option in the EEPROM config file may be used to indicate that the EEPROM should not be updated on this board. 
+### Reading current the EEPROM version
+```
+vcgencmd bootloader_version
+```
 
-Note that by default, updating the bootloader (automatically or manually) will retain any custom configuration options of the previous installed version. You can override the migration by manually updating and using the `-d` option with `rpi-eeprom-update`. This will force the updated bootloader to use its inbuilt defaults.
+## Updating the EEPROM configuration
+The bootloader EEPROM image contains a configuration file to define the boot behaviour (e.g. selecting between SD, network and USB boot).
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:49]: "an embedded configuration file"? :shrug:

File: hardware/raspberrypi/booteeprom.md [hunk 15]
```diff
@@ -2,82 +2,118 @@
...
+### Applying a saved configuration file
+The following command applies `boot.conf` to the latest available EEPROM image and uses `rpi-eeprom-update` to schedule an update when the system is rebooted.
 ```
-sudo rpi-eeprom-update -a
+sudo rpi-eeprom-config --apply boot.conf
 sudo reboot
 ```
 
-### Reading the current EEPROM configuration
+### Reading the configuration file from an EEPROM image
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:75]: Might be worth swapping things around so that "### Reading the configuration file from an EEPROM image" comes before "### Applying a saved configuration file" ?

File: hardware/raspberrypi/booteeprom.md [hunk 16]
```diff
@@ -2,82 +2,118 @@
...
+
+### recovery.bin
+At power on, the BCM2711 ROM looks for a file called `recovery.bin` in the root directory of the boot partition on the sd-card. If a valid `recovery.bin` is found then the ROM executes this instead of the SPI EEPROM image. This mechanism ensures that the bootloader SPI EEPROM can always be reset to valid image with factory default settings.
+
+#### EEPROM update files
+| Filename | Purpose |
+|----------|---------|
+| recovery.bin | VideoCore EEPROM recovery executable |
+| pieeprom.upd | Bootloader EEPROM image |
+| pieeprom.bin | Bootloader EEPROM image (same as pieeprom.upd) |
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:108]: Might be worth clarifying that you'd normally only have `pieeprom.upd` OR `pieeprom.bin`, and not both at the same time?

File: hardware/raspberrypi/booteeprom.md [hunk 17]
```diff
@@ -2,82 +2,118 @@
...
+| pieeprom.sig | The sha256 checksum of bootloader image (pieeprom.upd/pieeprom.bin) |
+| vl805.bin | The VLI805 USB firmware EEPROM image. Pi 4 revision 1.4 and newer do not have a dedicated SPI EEPROM for the VL805 USB controller |
+| vl805.sig | The sha256 checksum of vl805.bin |
+
+* If the bootloader update image is called `pieeprom.upd` then `recovery.bin` renames itself to `recovery.000` and resets the CPU. Since `recovery.bin` is no longer present the ROM loads the newly bootloader from SPI EEPROM and the OS is booted as normal.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:113]: "newly bootloader" -> "newly updated bootloader" ?

File: hardware/raspberrypi/booteeprom.md [hunk 18]
```diff
@@ -2,82 +2,118 @@
...
+* The `.sig` files should just contain the sha256 checksum (in hex) of the corresponding image file. Other fields should may be added in the future.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:114]: "should may be added" ?? (looks like the "should" is superfluous)
And this bullet point should probably be moved to after the next one?

File: hardware/raspberrypi/booteeprom.md [hunk 19]
```diff
@@ -2,82 +2,118 @@
...
+* If the bootloader update image is called `pieeprom.bin` the `recovery.bin` will stop after the update has completed. On success the HDMI output will be green and the green activity LED is flashed rapidly. Otherwise, the HDMI output will be read and an error code will be displayed via the activity LED.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:115]: the `recovery.bin` -> then `recovery.bin`

will be read -> will be red ?

File: hardware/raspberrypi/booteeprom.md [hunk 20]
```diff
@@ -2,82 +2,118 @@
...
+* The BCM2711 ROM does not support loading of recovery.bin from USB mass storage or TFTP. Instead, newer versions of the bootloader support a self-update mechanism where the SPI bootloader is able to reflash the SPI EEPROM itself. See `ENABLE_SELF_UPDATE` on the (bootloader configuration)[bcm2711_bootloader_config.md] page.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:116]: "loading of recovery.bin" could probably be changed to just "loading `recovery.bin`"

File: hardware/raspberrypi/booteeprom.md [hunk 21]
```diff
@@ -2,82 +2,118 @@
...
-# Enable it again
-sudo systemctl unmask rpi-eeprom-update
+If an update is available, you can install it using:
+```
+sudo rpi-eeprom-update -a
+sudo reboot
 ```
 
-The `FREEZE_VERSION` option in the EEPROM config file may be used to indicate that the EEPROM should not be updated on this board. 
+### Reading current the EEPROM version
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:43]: Reading current the EEPROM version -> Reading the current EEPROM version

File: hardware/raspberrypi/booteeprom.md [hunk 22]
```diff
@@ -2,82 +2,118 @@
...
 ```
-vcgencmd bootloader_version
+# -d instructs rpi-eeprom-update to use the EEPROM configuration inside new.bin instead of
+# of replacing it with the current configuration from vcgencmd bootloader.
+sudo rpi-eeprom-update -d -f new.bin
 ```
+For more information about advanced options please run `rpi-eeprom-update -h`.
+
+### recovery.bin
+At power on, the BCM2711 ROM looks for a file called `recovery.bin` in the root directory of the boot partition on the sd-card. If a valid `recovery.bin` is found then the ROM executes this instead of the SPI EEPROM image. This mechanism ensures that the bootloader SPI EEPROM can always be reset to valid image with factory default settings.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:101]: "reset to **a** valid image" ?

**Review by lurch — COMMENTED** [review]
I also noticed that some places talk about "bootloader configuration" and other places talk about "EEPROM configuration" - that's probably okay but I dunno if it _might_ cause confusion?

File: hardware/raspberrypi/booteeprom.md (hunk 11 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:6]: done

File: hardware/raspberrypi/booteeprom.md (hunk 12 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:30]: replaced with version

File: hardware/raspberrypi/booteeprom.md (hunk 13 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:31]: changed to upgrade. btw, we seem to duplicate apt-update upgrade all over the documentation. There probably ought to be a page which talks about that and just replaced all the duplicate comments with that

File: hardware/raspberrypi/booteeprom.md (hunk 14 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:49]: done

File: hardware/raspberrypi/booteeprom.md (hunk 15 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:75]: good point. done

File: hardware/raspberrypi/booteeprom.md (hunk 22 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:101]: done

File: hardware/raspberrypi/booteeprom.md (hunk 10 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:110]: done

File: hardware/raspberrypi/booteeprom.md (hunk 18 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:114]: done

File: hardware/raspberrypi/booteeprom.md (hunk 19 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:115]: done

File: hardware/raspberrypi/booteeprom.md (hunk 20 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:116]: done

File: hardware/raspberrypi/booteeprom.md (hunk 13 above)
Comment (lurch) [reply hardware/raspberrypi/booteeprom.md]: There's https://github.com/raspberrypi/documentation/blob/master/linux/software/apt.md and https://github.com/raspberrypi/documentation/blob/master/raspbian/updating.md ?

File: hardware/raspberrypi/booteeprom.md [hunk 23]
```diff
@@ -98,22 +98,23 @@ sudo rpi-eeprom-update -d -f new.bin
...
-At power on, the BCM2711 ROM looks for a file called `recovery.bin` in the root directory of the boot partition on the sd-card. If a valid `recovery.bin` is found then the ROM executes this instead of the SPI EEPROM image. This mechanism ensures that the bootloader SPI EEPROM can always be reset to valid image with factory default settings.
+At power on, the BCM2711 ROM looks for a file called `recovery.bin` in the root directory of the boot partition on the sd-card. If a valid `recovery.bin` is found then the ROM executes this instead of the SPI EEPROM image. This mechanism ensures that the bootloader SPI EEPROM can always be reset to a valid image with factory default settings.
 
 #### EEPROM update files
 | Filename | Purpose |
 |----------|---------|
 | recovery.bin | VideoCore EEPROM recovery executable |
 | pieeprom.upd | Bootloader EEPROM image |
-| pieeprom.bin | Bootloader EEPROM image (same as pieeprom.upd) |
+| pieeprom.bin | Bootloader EEPROM image - same as pieeprom.upd but with changes recovery.bin behaviour |
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:108]: Remove "with" ?

File: hardware/raspberrypi/booteeprom.md [hunk 24]
```diff
@@ -98,22 +98,23 @@ sudo rpi-eeprom-update -d -f new.bin
...
-* The BCM2711 ROM does not support loading of recovery.bin from USB mass storage or TFTP. Instead, newer versions of the bootloader support a self-update mechanism where the SPI bootloader is able to reflash the SPI EEPROM itself. See `ENABLE_SELF_UPDATE` on the (bootloader configuration)[bcm2711_bootloader_config.md] page.
+* If the bootloader update image is called `pieeprom.upd` then `recovery.bin` renames itself to `recovery.000` and resets the CPU. Since `recovery.bin` is no longer present the ROM loads the newly updated bootloader from SPI EEPROM and the OS is booted as normal.
+* If the bootloader update image is called `pieeprom.bin` the `recovery.bin` will stop after the update has completed. On success the HDMI output will be green and the green activity LED is flashed rapidly. Otherwise, the HDMI output will be red and an error code will be displayed via the activity LED.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:114]: the `recovery.bin` -> then `recovery.bin`

Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:114]: Probably makes sense to line "error code" to https://github.com/raspberrypi/documentation/blob/master/configuration/led_blink_warnings.md ?

File: hardware/raspberrypi/booteeprom.md [hunk 25]
```diff
@@ -98,22 +98,23 @@ sudo rpi-eeprom-update -d -f new.bin
...
+* The `.sig` files should just contain the sha256 checksum (in hex) of the corresponding image file. Other fields may be added in the future.
+* The BCM2711 ROM does not support loading `recovery.bin` from USB mass storage or TFTP. Instead, newer versions of the bootloader support a self-update mechanism where the SPI bootloader is able to reflash the SPI EEPROM itself. See `ENABLE_SELF_UPDATE` on the (bootloader configuration)[bcm2711_bootloader_config.md] page.
+* The temporary EEPROM updates files are automatically deleted by the `rpi-eeprom-update` service at startup.
```
Comment (lurch) [inline hardware/raspberrypi/booteeprom.md:117]: updates files -> update files

File: hardware/raspberrypi/booteeprom.md (hunk 13 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md]: Possibly updating.md - it's a bit wordy though - I think most people just want a something that they can run without thinking too hard. Something to think about for the future though

File: hardware/raspberrypi/booteeprom.md (hunk 24 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:114]: added. shouldn't there be some sort of wiki style autolinking for docs?

File: hardware/raspberrypi/booteeprom.md (hunk 25 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:117]: done

File: hardware/raspberrypi/booteeprom.md (hunk 23 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:108]: done

File: hardware/raspberrypi/booteeprom.md (hunk 24 above)
Comment (timg236) [reply hardware/raspberrypi/booteeprom.md:114]: done

**Review by lurch — APPROVED** [review]
LGTM but @JamesH65 might want to check it too?

**Comment by JamesH65** [comment]
Looks great, merging.
